Part2_LinuxKernelBootOptions.html

```diff
@@ -342,15 +342,15 @@ <h3>noapic</h3>
 ERR:         0
 MIS:         0</pre></source>
 
-	<p>As we see, all IO-APIC-* interrupts have turned into XT-PIC-XT-PIC, and all of these interrupts have been routed to the CPU0 only. MSI interrupts in the other case have remained unchanged and go to all of the CPU0-3.<p/>
+	<p>As we see, all IO-APIC-* interrupts have turned into XT-PIC-XT-PIC, and all of these interrupts have been routed to CPU0 only. MSI interrupts on the other hand have remained unchanged and go to all of the CPUs.<p/>
 
 	<h3>nolapic</h3>
 
-	<p>This kernel boot option disables LAPIC. MSI interrupts can't work without LAPIC, I/O APIC can't work without LAPIC. Because of that all the interrupts from the devices in the system will go to PIC controller, and it works with CPU0 only. And without LAPIC the rest of the CPUs can't even work in the system.<p/>
+	<p>This kernel boot option disables LAPIC. MSI interrupts can't work without LAPIC, and I/O APIC can't work without LAPIC either. Because of this all the interrupts from the devices in the system will go to the PIC controller, and it works with CPU0 only, since without LAPIC the rest of the CPUs can't even work in the system.<p/>
```

Comment:
Can it be like in the conclusion?
"All of the device interrupts can only go to the PIC, and it works with the CPU0 only. And without LAPIC the rest of the CPUs besides CPU0 won't work."

I think it is not right to use "since", because the first sentence just states a fact, that PIC is only connected to the CPU0, and it is not because of the lack of APIC, it is just how PIC is routed

Reply:
Yep that sounds good.